torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -1413,28 +1413,28 @@ static StableIValue from_ivalue(
     case c10::TypeKind::TensorType: {
       AtenTensorHandle ath = torch::aot_inductor::new_tensor_handle(
           std::move(const_cast<at::Tensor&>(ivalue.toTensor())));
-      return from(ath);
+      return torch::stable::detail::from(ath);
```

Comment:
Just add `using namespace torch::stable::detail` on line 1411 and don't make any changes later on